omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap.validator.rb

```diff
@@ -0,0 +1,238 @@
+#
```

Comment:
or even `bootstrap_preflight_validator.rb` to match the class (I know there's some oddity with the alphabet sorting).

@marcparadise perhaps the `.` vs `_` is why you were seeing odd errors this morning?